### ISSUE TITLE
Problems with the delete key when using the orca screen reader

### ISSUE THREAD

**Issue by jvesouza** [microsoft/vscode#103262]
CC: @isidorn 
Issue Type: <b>Bug</b>

1. Launch orca.
1. Launch VScode insiders and open a file to edit.
3. Type something and try to delete what was typed using the delete key.

Orca should announce the character that was to the right of the removed character but instead it is muted.

According to @joanmarie this issue is:
https://bugs.chromium.org/p/chromium/issues/detail?id=1065534
and the fix is:
https://chromium.googlesource.com/chromium/src/+/2469d759e3

Any chance of cherry-pick this fix?

VS Code version: Code - Insiders 1.48.0-insider (8c426f9f3b6b18935cc6c2ec8aa6d45ccd88021e, 2020-07-23T05:32:52.579Z)
OS version: Linux x64 5.7.10-arch1-1

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz (4 x 2719)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: unavailable_off<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 1, 1|
|Memory (System)|9.60GB (0.15GB free)|
|Process Argv|--no-sandbox --crash-reporter-id 8d5a3738-d241-4b34-b208-057882a7b877|
|Screen Reader|yes|
|VM|0%|
|DESKTOP_SESSION|gnome-xorg|
|XDG_CURRENT_DESKTOP|GNOME|
|XDG_SESSION_DESKTOP|gnome-xorg|
|XDG_SESSION_TYPE|x11|
</details>
We have written the needed data into your clipboard because it was too large to send. Please paste.

**Comment by isidorn** [comment]
@jvesouza thanks for filling this along with the link to the upstream issue and the fix.
@deepak1556 is it possible to cherry-pick the fix, does this make sense to you?

**Comment by deepak1556** [comment]
yup its possible, the linked fix went into chromium ` 84.0.4109.1` , I will backport it to Electron 9 which is our current insiders based on chromium 83.

**Comment by jvesouza** [comment]
Problem fixed, I believe we can close this bug. Thank you all.

**Comment by isidorn** [comment]
@jvesouza thanks for letting us know. We plan to close this once we definetly decide to ship Electron 9 as part of stable. Thus we have assigned the `fixed-in-electron-9` label. 
We hope the Electron 9 will happen this milestone.

**Comment by isidorn** [comment]
Seems like we will shipt with Electron 9, thus closing this optimistically.
